### PR TITLE
Addition of Elden Ring Plugin

### PR DESCRIPTION
**Pull request by Roselyyn**
Plugin version of the Elden Ring 2022 april fools addition

### CODE REVIEW

File: testing/EldenRing/EldenRing.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Author": "Roselyn",
+  "Name": "Elden Ring April Fools",
+  "Punchline": "Recreation of the Elden Ring April Fools joke.",
+  "Description": "Currently under development with extra features, should currently function as the April Fools joke did.", //Add /commands later
```
Comment (MidoriKami) [inline testing/EldenRing/EldenRing.json]: Possible Error in json file

Comment (Roselyyn) [reply testing/EldenRing/EldenRing.json]: Fixed it, forgot I left it there

**Comment by goaaats** [comment]
Did you test if your packed build works? You're missing NAudio DLLs from the zip, they're not gonna be packed with Dalamud anymore - just making sure

**Comment by reiichi001** [comment]
This PR needs to be updated as the actual plugin source has been updated a bit.

I made sure NAudio's being packaged with builds. 

Once it's updated, we can re-review it and then get it merged.

**Comment by Roselyyn** [comment]
Changed it to be the up to date version with NAudio

**Pull request merged**